client/signup/steps/user/index.jsx

```diff
@@ -225,6 +228,10 @@ export class UserStep extends Component {
 			} );
 		}
 
+		if ( 'onboarding-dev' === flowName ) {
```

Comment:
I get worried when I see lots of this kind of thing peppered around our code that we are increasing complexity by forking the number of variations. I'd rather we either made this change for every flow, or created a new step, but maybe this is only a temporary measure?

Reply:
Yes, I believe this is only a temporary measure, since eventually onboarding-dev becomes our new main flow; we're trying to avoid updating the current main flow with this change.